odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+import odl
+from odl.contrib.datasets.util import get_data
+from scipy.misc import imresize
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'building', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images'
```

Comment:
this needs to be more specific, e.g. `images_cambridge` to avoid collisions with other image libraries in the future